src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -34,14 +35,15 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+    public void addToCart(Authentication auth, @RequestParam Long movieSessionId) {
+        User userByEmail = userService.findByEmail(auth.getName()).get();
         shoppingCartService.addSession(
-                movieSessionService.get(movieSessionId), userService.get(userId));
+                movieSessionService.get(movieSessionId),userByEmail);
```

Comment:
```suggestion
                movieSessionService.get(movieSessionId), userByEmail);
```
I wonder how checkstyle allowed this